module/components/Badge/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Badge as ReactstrapBadge } from 'reactstrap';
+
+const propTypes ={
+    /**
+     * Useful to extend the style applied to components.
+     */
+    classes: PropTypes.string,
+    /**
+     * The color of the component. It's using the theme palette when that makes sense.
+     */
+    color: PropTypes.string,
```

Comment:
shall we make it `PropTypes.oneOf(['primary', 'secondary', 'success', 'info', 'warning', 'danger']),` so its testable?

Reply:
No, we are allowing to add any color

Reply:
@dakshika, im agree with @thusithak too. I think we should only keep the colors that we support by the framework right? because this will add the css class `badge-*`. So nothing will happen if users use a none supported color string here. If users want to add a color other than what we support, they can use the `className` property to add a custom class.